src/figuresClasses.ts

```diff
@@ -1,19 +1,77 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
```

Comment:
it shouldn't be any string

above this interface define types (enum or union) for `shape` and` color`